assets/android/kotlin/res/drawable/iconThread.xml

```diff
@@ -0,0 +1,13 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
The path `assets/android/kotlin/res/drawable/iconThread.xml` is not correct if we ever want to build an aar from the folder. We can deal with that later.

I would propose to use the path

`assets/android/res/drawable/iconThread.xml`

for now.

Reply:
Also on the file name I would stick to Android convention, is is possible to have something like `ic_thread.xml`?

At least avoid using capital letters.

Reply:
I mentioned some of this above, although I didn't notice the issue with capital letters. However, the XML vector drawable is already in `res/drawable`, so I think there's no need to move it to assets, as it would probably make using it more difficult?

Reply:
`kotlin/res` was weird to me, but if the objective is to generate a "buildable" Android gradle module, more changes will be necessary.

Reply:
Done!
A follow up project of mine will be to make this a gradle module, but that requires a bit more work than needed for this PR

Reply:
I've snake cased the file name and replaced `icon` with `ic`